S/sshpass/build_tarballs.jl

```diff
@@ -0,0 +1,41 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "sshpass"
+version = v"1.10.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://downloads.sourceforge.net/project/sshpass/sshpass/$(version.major).$(version.minor)/sshpass-$(version.major).$(version.minor).tar.gz",
+                  "ad1106c203cbb56185ca3bad8c6ccafca3b4064696194da879f81c8d7bdfeeda"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd sshpass-*
+
+# The default configure.ac includes a malloc check that causes failures on
+# ARM/OSX/musl: https://github.com/maxmind/libmaxminddb/pull/152
+# Submitted upstream: https://sourceforge.net/p/sshpass/patches/16/
+atomic_patch -p1 ../patches/configure-ac.patch
+
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
```

Comment:
I'm confused: you patch configure.ac and then immediately run configure without regenerating it? I'm missing how this works.

Reply:
Right I see what was happening, `make` saw the changed `configure.ac` file and was regenerating `configure` and running it again: https://buildkite.com/julialang/yggdrasil/builds/14255#0192c5d5-a395-4527-974b-c31ea530086c/660-851

I added a call to `autoreconf -i` in 0c8223c8 so that should be fixed now.